packages/core/src/V2/Node.ts

```diff
@@ -0,0 +1,41 @@
+import { Observable, Subscription, SubscriptionLike } from 'rxjs';
+
+export interface INodePorts {
```

Comment:
One of the best programmers I know scolded me big time for using I/Interface in identifiers 😄 we cant do it! https://stackoverflow.com/questions/68503504/what-is-the-naming-convention-for-interfaces-and-classes-in-typescript